src/client.cpp

```diff
@@ -736,7 +736,7 @@ Status Client::Impl::uploadObjectStream(id_t accountId, id_t instrumentId,
         return makeNetworkError();
     }
 
-    if (cs->getResponseCode() != 201)
+    if (cs->getResponseCode() != 201 && cs->getResponseCode() != 200)
```

Comment:
At this moment uploadObjectStream allows 200 & 201, but at other places, like uploadBackground only 201 allowed; looks like mess.
2xx code is success code,so it is wrong to say that errors happens if code equal for ex. 204. I think in this case better to write just warning: "Data processed but response is unusual".